emission/ce186/data.csv

```diff
@@ -0,0 +1,39 @@
+30
```

Comment:
What does data.csv encode? Some comments in a README or in a pull request would be useful.